store/tikv/coprocessor.go

```diff
@@ -93,16 +93,22 @@ func (c *CopClient) Send(ctx context.Context, req *kv.Request, vars *kv.Variable
 		it.respChan = make(chan *copResponse, it.concurrency)
 		it.sendRate = newRateLimit(it.concurrency)
 	}
+	workersCond := sync.NewCond(&sync.Mutex{})
+	it.actionOnExceed = newTaskRateLimitAction(it.sendRate, workersCond)
+	if it.memTracker != nil {
```

Comment:
Is it not right here?
Maybe we need to pass `sctx.GetSessionVars()` instead of `sctx.GetSessionVars().KVVars` here?
Thus we can invoke `sctx.GetSessionVars().StmtCtx.MemTracker. FallbackOldAndSetNewAction `